content/posts/continuous_training/building_training_pipeline.md

```diff
@@ -130,6 +181,7 @@ So after training our model, making sure it outperforms the baseline and gaining
 </figure>
 
 Bear in mind that our model is simply an artifact, a file that contains the weights of the model. In order to make predictions, we need to wrap it in an API. We will use the <a href="https://pytorch.org/serve/">`torchserve`</a> library. The process goes as follows:
+
 1. **Model Archiver**: we need to create a `.mar` file that contains the model and the code to run it: 
```

Comment:
Code formatting for the config.properties file looks weird

Reply:
Also the code formatting in the code section is not working as expected:
Note: we have reformated the `config.properties` for clarity. However, the model_snapshot must be a single line without spaces, as in the original [file](https://github.com/InakiRaba91/animal_classifier/blob/main/config.properties).